fe/fe-core/src/main/java/com/starrocks/privilege/DefaultAuthorizationProvider.java

```diff
@@ -155,6 +161,9 @@ public PEntryObject generateObject(ObjectType objectType, List<String> objectTok
             case RESOURCE_GROUP:
                 return ResourceGroupPEntryObject.generate(mgr, objectTokens);
 
+            case STORAGE_VOLUME:
```

Comment:
built-in role `db_admin` should have any priv on any storage volumn?

Reply:
> built-in role `db_admin` should have any priv on any storage volumn?

It should have all of the priv on storage volumes